resources/src/books.php

```diff
@@ -181,7 +180,7 @@ public function book_info($id)
             }
             $stmt1 = $this->pdo->query('SELECT title FROM Book where Book.id="'.$row2["bid"].'"');
             
-            while($row1=$stmt1->fetch(PDO::FETCH_ASSOC))
+            while($row1=$stmt1->fetch())
```

Comment:
same here

Reply:
@guptahemant  I have set it default in the config while declaring the PDO object in dbConn.php so it needed?